static/src/javascripts/projects/common/modules/experiments/tests/contributions-moment-banner-environment.js

```diff
@@ -6,21 +6,21 @@ import { canShowBannerSync } from 'common/modules/commercial/contributions-utili
 import { getSync as geolocationGetSync } from 'lib/geolocation';
 
 const geolocation = geolocationGetSync();
-const campaignId = 'environment_campaign';
+const campaignId = 'climate_pledge_2019';
 
 const defaultLeadSentence =
     'The climate emergency is the defining issue of our times. ';
 const defaultCopy =
-    'This is The Guardian’s pledge: we will be truthful, resolute and undeterred in pursuing our journalism on the environment. We will give global heating, wildlife extinction and pollution the urgent attention they demand. Our independence means we can interrogate inaction by those in power. It means our reporting will always be grounded in scientific facts, never in commercial or political interests. Support from our readers makes this work possible.';
+    'This is the Guardian’s pledge: we will be truthful, resolute and undeterred in pursuing Guardian journalism on the environment. We will give global heating, wildlife extinction and pollution the urgent attention they demand. Our independence means we can interrogate inaction by those in power. It means Guardian reporting will always be driven by scientific facts, never by commercial or political interests. Support from our readers makes this work possible.';
```

Comment:
Just curious - but isn't this inconsistent with the way we refer to "The Guardian" in other places...?

Reply:
I've only seen it as 'The Guardian' as well... @ionamckendrick is this an editorial style change?